articles/liquity/Asset Risk Assessment - bLUSD.md

```diff
@@ -0,0 +1,203 @@
+
+The amplified yield comes from the yield generated through the funds in the pending and permanent bucket. Currently, this yield amplification is 10.53 as per the below data ([Liquity.app](https://liquity.app/#/bonds)). The amplification would reduce in the case where the Pending bucket holds fewer funds compared to other buckets.
+
+![LUSD bonds statistics November 2nd 2022](https://lh5.googleusercontent.com/MsqVvA0LBqmYVPRKlVjIC3kahF6jrkEuMwtBdFgkEqhXTJ2odOshXnclbE6Pdp9tLwyb6ET7hoSA80oUDNF7nLlQ-tfPovQWY9FH5WTuoOADWz-J9WbiHGWJxiqxVN1u8_-IKYFzGpagc3o5NF14kmQ06KpoHWyVsjk6P1vVzy-SgqEOglICux0mDAW-Gg)
+
+As of Nov 2nd, 2022
+
+The market price of bLUSD can be influenced by this ability to amplify the yield amount in the pending bucket i.e. funds that are bonded but have yet to chicken_in or chicken_out.
+
+The impact of the market price of bLUSD will be discussed further in the article.
+
+## bLUSD accrual mechanism
+
+A user who has bonded LUSD will get nothing but a promise of redeeming accrued bLUSD at the time of chicken-in, foregoing the bonded LUSD. This accrual of bLUSD depends on the variables like time, bonded amount, chicken in fees, accrual parameter (alpha), and redemption price of bLUSD.
+
+Here is the equation:
+
+$$
+s(t) = \frac {b}{p_r}(\frac {t}{t + \alpha})
+$$
+
+$s(t)$ = Virtual accrual of bLUSD
+
+$b$ = Amount of LUSD bonded
+
+$\alpha$ = Accrual parameter
+
+$t$ = Time in seconds
+
+$p_r$ = Redemption price (LUSD in Reserve Bucket / bLUSD supply)
+
+The amount of LUSD bonded (b) is taken into consideration after reducing the 3% chicken in fees from the actual bond value for the calculation of bLUSD accrual. The accrual rate might vary based on the current redemption price (LUSD in the Reserve bucket / bLUSD supply).
+
+Moreover, there is a function where alpha is reduced to accelerate the bLUSD accrual rate. This is triggered when the weighted average bond age goes beyond the target age.
+
+![Controller logic for triggering the accrual parameter](https://lh4.googleusercontent.com/k3o7OuY1SC95kNCOCsNumC7Lx1gjbdQkyLl4HCL4-oizhUWdB9gK93Hj60Hd0ha8w_7DTDtEdPsZn3WMFSDNIDEX1_kTVOLRmtMwu2P2pOkzYM15ymjMfkJcKKtbvuQLboZMUjxHGmz-FPBTt2MmJJV-6TjarZAhM_kGG16EcnHbRhN_AIHHey4C_2DMfg)
+
+Lowering the alpha would cause the accrual of bLUSD to speed up thus bringing the break-even point and the optimal rebonding time close. This will in turn cause the average bond age to drop which is why the controller is used.
+
+## A consequence of lowering the alpha
+
+A major reason for greater bond age is people not willing to take an action (CI or CO), in other words, people are not convinced with the currently accrued bLUSD is profitable enough to chicken_in. When the [current weighted average bond age](https://dune.com/queries/1460655) is greater than the target age which is 15 days, the controller would lower the alpha.
```

Comment:
"not convinced with" => "not convinced that"

Reply:
Does this sound/read well?

A major reason for greater bond age is people not willing to take an action (Chicken in or Chicken out), in other words, people not convinced enough that the bLUSD accrued so far is profitable enough to chicken in. When the [current weighted average bond age](https://dune.com/queries/1460655) is greater than the target age which is 15 days, the controller would lower the alpha.

Reply:
It sounds fine, though you give a reason for why people would refrain from chickening in but not for why they wouldn't chicken out?

Reply:
I just wanted to lead the readers to make them understand in what condition will the controller be triggered.

> why people wouldn't chicken out?

A user might not choose to chicken out if they feel the bLUSD market price can go up or maybe due to sunk cost fallacy. They might chicken out as well; it will depend on the market condition, their bond age etc.

I believe the above would not add much to what I want to convey (the impact of lowering the accrual parameter) so I didn't include it.